CONTRIBUTING.md

```diff
@@ -1,66 +1,57 @@
 # Contributing
 
-Contributions are welcomed! 
+Contributions are welcomed!
 
-When contributing to this repository, please first discuss the change you wish to make via GitHub
-issue before making a change.  This saves everyone from wasted effort in the event that the proposed
-changes need some adjustment before they are ready for submission.
+When contributing to this repository, please first discuss the change you wish to make via GitHub issue before making a change. This saves everyone from wasted effort in the event that the proposed changes need some adjustment before they are ready for submission.
 
 ## Pull Request Process
 
 1. Fork the repo, push your commits to a branch of your fork, open the PR.
 2. Make sure you update the README.md where relevant.
-3. Project maintainers will squash-merge Pull Requests once they are happy.  There may be one or more
-   cycles of feedback on the PR before they are satisfied.
+3. Project maintainers will squash-merge Pull Requests once they are happy. There may be one or more cycles of feedback on the PR before they are satisfied.
 
 ## Build and run tests locally
 
 The software is written in [Scala](https://scala-lang.org/) and is built with [SBT](http://www.scala-sbt.org/).
 
-The project is built and released for Scala versions 2.12 and 2.13.
-To compile and test both versions run `sbt +test`.
+The project is built and released for Scala versions 2.12 and 2.13. To compile and test both versions run `sbt +test`.
 
 ## Performing a release (for project maintainers)
 
-1. Follow the [sbt-bintray publishing guidelines](https://github.com/sbt/sbt-bintray#publishing) to ensure you are
-   authenticated with Bintray.  Note that you need to be a member of the `sky-uk` Bintray organization. 
-2. Run `sbt release` to perform the release of the binaries to Bintray.
-3. Check you are happy with the draft publication of the new version [here](https://bintray.com/sky-uk/oss-maven/kafka-topic-loader)
-   and, if so, run `sbt bintrayRelease` to make the new version publicly accessible.
+Merging a PR to master will create a snapshot release in the [Sonatype snapshot repository](https://s01.oss.sonatype.org/content/repositories/snapshots/uk/sky/).
+
+To create a new stable release, you must tag the head of master and push it to Github.
+
+```bash
```

Comment:
instead of this, we should create a GitHub release instead (which will create the tag you specify on the UI)

Reply:
Creating a tag is a manual task anyway, so I'm happy to use the UI to create a release (and have that create the tag).

Reply:
Yeah either works, as long as the tag is created on the remote the job will run

Reply:
[`fb94c8b` (#74)](https://github.com/sky-uk/kafka-topic-loader/pull/74/commits/fb94c8bb11f80f9876f4613f4be419ac1696f7e2)